JSBrowser/js/components/address-bar.js

```diff
@@ -137,9 +137,41 @@
         style.display = isHidden ? "inline-block" : "none";
     };
 
+    this.colorizeAddress = address => {
+        let HTML = '';
+        // Parse URL using an <a> element
+        let a = document.createElement('a');
```

Comment:
For URL parsing you can use the [Windows.Foundation.Uri](https://www.google.com/search?q=Windows.Foundation.Uri&oq=Windows.Foundation.Uri&aqs=chrome..69i57j0l4.656j0j7&sourceid=chrome&es_sm=93&ie=UTF-8) WinRT API.

We have the `URI` const set [here](https://github.com/DJDavid98/JSBrowser/blob/4f2d59a04b09ac1cbcccb6cbffa7e4149df393c3/JSBrowser/js/components/address-bar.js#L7).
You can see a usage example [here](https://github.com/DJDavid98/JSBrowser/blob/4f2d59a04b09ac1cbcccb6cbffa7e4149df393c3/JSBrowser/js/components/address-bar.js#L7)